mnd/mnd/registry/patients/mnd_admin_forms.py

```diff
@@ -158,13 +158,10 @@ def save(self, commit=True):
         if instance and instance.pk:
             db_instance = PrimaryCarer.objects.get(pk=instance.pk)
             email_changed = db_instance.email != email
-            existing_user = CustomUser.objects.filter(username=db_instance.email, is_active=True).exists()
-            carer_instance_update = email_changed and existing_user
-            if carer_instance_update:
+            if email_changed:
```

Comment:
Can we use `self.changed_data` instead? This method has become very hairy, is it possible to simplify?

Reply:
Updated.